tests/test_views.py

```diff
@@ -1,15 +1,32 @@
 import pytest
-
+from django.urls import reverse
+from collections import OrderedDict
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+
+    expected_return_input_string = '123 main st chicago il'
```

Comment:
Why do you need this assigned separately from `address_string` above?

Reply:
Yeah, I understand why that is confusing. I made that choice for my own visual preference. When writing my tests, I like to compartmentalize what I am giving to the test and what I expect in return. 

Since there is an intended overlap (the expected return string should be the given address string), my `expected_return_input_string` is redundant and does not need to be assigned separately. I'll fix that up!